composer.json

```diff
@@ -44,13 +44,18 @@
   "require": {
     "composer/installers": "^1.9",
     "cweagans/composer-patches": "^1.6.5",
+    "drupal/antibot": "^1.4",
+    "drupal/better_exposed_filters": "^5.0@beta",
     "drupal/better_normalizers": "^1.0@beta",
     "drupal/block_content_permissions": "1.x-dev",
+    "drupal/block_field": "^1.0@RC",
```

Comment:
Page manager, layout builder and this one looks overkill

Reply:
Removed